pack/fhv/fhvc.json

```diff
@@ -165,10 +446,266 @@
+    "name": "Molting Hybrid",
+    "pack_code": "fhvc",
+    "position": 144,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (west).\nAloof. Patrol (The Farmhouse).\nMolting Hybrid ignores barriers.\n<b>Forced</b> – When Molting Hybrid would take any amount of damage: Reduce that amount to 1.",
+    "traits": "Creature. Monster. Mutated.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10645a",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 22,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 4,
+    "illustrator": "Helge C. Balzer",
+    "name": "Lupine Hybrid",
+    "pack_code": "fhvc",
+    "position": 145,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (north).\nRetaliate. Patrol (The Farmhouse).\n<b>Forced</b> – After Lupine Hybrid attacks: Heal 1 damage from it.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10645b",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 23,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 4,
+    "illustrator": "Helge C. Balzer",
+    "name": "Lupine Hybrid",
+    "pack_code": "fhvc",
+    "position": 145,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (east).\nRetaliate. Patrol (The Farmhouse).\n<b>Forced</b> – After Lupine Hybrid attacks: Heal 1 damage from it.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10645c",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 24,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 4,
+    "illustrator": "Helge C. Balzer",
+    "name": "Lupine Hybrid",
+    "pack_code": "fhvc",
+    "position": 145,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (south).\nRetaliate. Patrol (The Farmhouse).\n<b>Forced</b> – After Lupine Hybrid attacks: Heal 1 damage from it.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10646",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 25,
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "flavor": "I suppose that, like any goat, it eats everything.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
```

Comment:
```suggestion
    "flavor": "I suppose that, like any goat, it eats everything.\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

Reply:
Generally speaking, we've erred towards keeping it simple and just using a dash and underline for titles. So I'd prefer for this for consistency across the db.